### PR TITLE
feat(webpack): Support open browser automatically in the developmen…

### PR DESCRIPTION
**Pull request by GayeChen**
…t environment

## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
- [ ] double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-boilerplate/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

**Comment by Mensae** [comment]
Sorry to say but this feature always annoyed me when I still used create-react-app. Ending up closing new tabs all the time and even more annoying when you have multiple browsers open where a browser gets focus that you are not per sé developing in.
If others like this feature then I vote to make that this can be dissabled.

File: server/port.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 const argv = require('./argv');
 
-module.exports = parseInt(argv.port || process.env.PORT || '3000', 10);
+module.exports = parseInt(argv.port || process.env.PORT || '8000', 10);
```
Comment (sourcesoft) [inline server/port.js]: Why changing the port?

Comment (GayeChen) [reply server/port.js]: The  port 3000  likely to be occupied....

**Comment by GayeChen** [comment]
Thanks for your reply, indeed 
I like the template so much， Can official website provide some successful cases of using this template
------------------ 原始邮件 ------------------
发件人: "Johan Meester";<notifications@github.com>;
发送时间: 2017年12月27日(星期三) 晚上9:08
收件人: "react-boilerplate/react-boilerplate"<react-boilerplate@noreply.github.com>;
抄送: "情书"<1580850542@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [react-boilerplate/react-boilerplate] feat(webpack): Support open browser automatically in the developmen… (#2084)




Sorry to say but this feature always annoyed me when I still used create-react-app. Ending up closing new tabs all the time and even more annoying when you have multiple browsers open where a browser gets focus that you are not per sé developing in.
 If others like this feature then I vote to make that this can be dissabled.

File: package.json [hunk 2]
```diff
@@ -286,6 +286,7 @@
     "node-plop": "0.7.0",
     "null-loader": "0.1.1",
     "offline-plugin": "4.8.1",
+    "open-browser-webpack-plugin": "^0.0.5",
```
Comment (avdeev) [inline package.json]: Use exact version

**Comment by mxstbr** [comment]
Deploy preview for *react-boilerplate* failed.

Built with commit 30558b3ef1a8e2bb631ec933bcff179f4f51ab39

https://app.netlify.com/sites/react-boilerplate/deploys/5ac1d8c5be40f1741151b3b5

**Pull request closed without merging**